stable/elastabot/values.yaml

```
@@ -21,6 +21,9 @@ elasticsearch:
 # Elastalert settings. Elastalert chart information is available here:
 # https://github.com/kubernetes/charts/tree/master/stable/elastalert
 elastalert:
+  # Index prefix for elastalert
+  index: "elastalert"
```

```suggestion
  index: "elastalert_status"
```
Shouldn't the previous default value be kept? Isn't it a breaking change otherwise?

---

Unfortunately, the default index was based on an old, flawed Elastalert release that had a wrong default index as well. This caused indexes to be named `elastalert_status_status`. That glitch was fixed many months ago and their default is now corrected, so I wanted to keep this companion chart in sync with those defaults. I expect that the majority of users of this chart would have had to override this default in order for Elastabot to have worked correctly with recent versions of Elastalert.